main.tf

```diff
@@ -129,6 +131,28 @@ data "aws_iam_policy_document" "default" {
     }
   }
 
+  dynamic "statement" {
+    for_each = var.allow_ssl_requests_only ? [1] : []
+
+    content {
+      sid       = "ForceSSLOnlyAccess"
```

Comment:
I feel like the language of the Sid should match the language of the feature flag

Reply:
@osterman updated